packages/ra-ui-materialui/src/input/ArrayInput/SimpleFormIterator.tsx

```diff
@@ -166,19 +190,39 @@ export const SimpleFormIterator = (props: SimpleFormIteratorProps) => {
                         </SimpleFormIteratorItem>
                     ))}
                 </ul>
-                {!disabled && !disableAdd && (
-                    <div className={SimpleFormIteratorClasses.add}>
-                        {cloneElement(addButton, {
-                            className: clsx(
-                                'button-add',
-                                `button-add-${source}`
-                            ),
-                            onClick: handleAddButtonClick(
-                                addButton.props.onClick
-                            ),
-                        })}
-                    </div>
-                )}
+                <div className={SimpleFormIteratorClasses.line}>
```

Comment:
This class name is used for `li` in SimpleFormIteratorItem, you're reusing it in another context. This means that developers can't change one and not the other.

Reply:
New class name called `button` is created.
This goal is for aligning row its children

commit 1125d555d9b6cfaf349e741f163833d37d11405d